addons/l10n_ar/views/report_invoice.xml

```diff
@@ -272,6 +272,30 @@
             <attribute name="t-call">l10n_ar.document_tax_totals</attribute>
         </t>
 
+        <div id="qrcode" position="after">
+            <!-- RG 5614/2024: Show ARCA VAT and Other National Internal Taxes -->
+            <t t-set="l10n_ar_custom_tax_summary" t-value="o._l10n_ar_get_invoice_custom_tax_summary_for_report()"/>
+            <div class="l10n_ar_tax_details col-sm-7 col-md-6" t-if="l10n_ar_tax_details">
+                <table class="table table-sm table-borderless" style="page-break-inside: avoid;">
+                    <th class="border-black" style="border-bottom: 1px solid" colspan="2">
+                           Fiscal Transparency Regime for the Final Consumer (Law 27.743)
+                    </th>
+                    <t t-foreach="l10n_ar_custom_tax_summary" t-as="detail">
+                        <tr>
+                            <td class="text-end"><strong t-esc="detail['name']"/></td>
+                            <td class="text-end">
+                                <span
+                                    class="oe_subtotal_footer_separator"
+                                    t-out="detail['tax_amount_currency']"
+                                    t-options='{"widget": "monetary", "display_currency": currency}'
```

Comment:
For some reason, using this widget is returning an error (see gist https://gist.github.com/zaoral/c01d3268aae343fba3cfa317630e619b)
However, we don't want/need to print the currency symbol. This is already added in the label of the tax group that is shown, removing the t-option will solve the problem
![image](https://github.com/user-attachments/assets/25533b5e-5ed6-44ba-8ad8-908f7027197d)